### ISSUE TITLE
Pinion not compatible with latest version of PcbDraw

### ISSUE THREAD

**Issue by minecraft2048** [yaqwsx/Pinion#32]
With latest pip version of Pinion and PcbDraw I got this error:
```
n310-drive@n310 /r/n/harry-kicad> pinion generate --style rendered --board harry-kicad.kicad_pcb --specification pinout/test.yaml ./pinout
Traceback (most recent call last):
  File "/home/n310-drive/.local/bin/pinion", line 8, in <module>
    sys.exit(cli())
  File "/usr/lib/python3/dist-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/home/n310-drive/.local/lib/python3.10/site-packages/pinion/ui.py", line 77, in generate
    from pinion.generate import generate
  File "/home/n310-drive/.local/lib/python3.10/site-packages/pinion/generate.py", line 14, in <module>
    from pcbdraw.pcbdraw import svg2ki
ModuleNotFoundError: No module named 'pcbdraw.pcbdraw'
```
But if I install PcbDraw from commit https://github.com/yaqwsx/PcbDraw/commit/1071f3ba48390abc18794c94c854a11541c87045 and latest Pinion there's no more `ModuleNotFoundError`

EDIT: this commit: https://github.com/yaqwsx/PcbDraw/commit/fead173fb91041c5bc4e809aaee0574b13773e9f#diff-5e0ba9a3dd07d0b08ab5328458d98bd2b5bff2246b5be75cc4ce9f205ad6f206 deleted pcbdraw.pcbdraw module

**Comment by yaqwsx** [comment]
There was a wrongly specified version of PcbDraw in the `setup.py`. We required `>=0.8` instead of `~=0.8`. This allowed the users to use Pinion with PcbDraw v1 for which it has not been adapted yet.

I pushed the initial changes to ensure compatibility in fcd0f5aa39bed8fd091ff8d82c546f0bec3400ba (there's still more work to be done to bring in the new functionality of v1, but at the moment it should work as used to).

**Comment by yaqwsx** [comment]
There was a wrongly specified version of PcbDraw in the `setup.py`. We required `>=0.8` instead of `~=0.8`. This allowed the users to use Pinion with PcbDraw v1 for which it has not been adapted yet.

I pushed the initial changes to ensure compatibility in fcd0f5aa39bed8fd091ff8d82c546f0bec3400ba (there's still more work to be done to bring in the new functionality of v1, but at the moment it should work as used to).
